package/boot/arm-trusted-firmware-mvebu/Makefile

```diff
@@ -32,7 +32,7 @@ define Trusted-Firmware-A/espressobin-512mb
   BUILD_DEVICES:=globalscale_espressobin
   UBOOT:=espressobin
   DDR_TOPOLOGY:=0
-  CLOCKSPRESET:=CPU_800_DDR_800
+  CLOCKSPRESET:=CPU_1000_DDR_800
```

Comment:
Will this update to 1GHz CPU clock work stable without the kernel changes?

Reply:
Then it will actually never run at 1GHz. It claims it does, but it's stuck at 800MHz. I'm running debian with this patch (and its unpatched kernel) stable for weeks now.

It's unstable with an older kernel patch that just fixes it reaching 1GHz, without the AVS bump. The new kernel patchset fixes both, see the link in this patch.

Reply:
ok, so with current openwrt kernel it would just run at 800MHz anyway and there are some patches for mainline kernel which make it work stable with 1GHz and there is an older patchset which was not in OpenWrt or mainline which makes it unstable. Is this correct?

Reply:
Correct. What I linked to is the proper patch set. The older patch was only used by armbian afaik.